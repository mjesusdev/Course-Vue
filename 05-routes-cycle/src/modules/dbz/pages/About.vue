<template>
    <h3>About DBZ</h3>
</template>