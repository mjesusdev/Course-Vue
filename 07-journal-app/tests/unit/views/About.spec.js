import About from '@/views/About'
import { shallowMount } from '@vue/test-utils'

describe('Test in About View', () => {
    test('should run the component correctly', () => {
        const wrapper = shallowMount(About);
        expect( wrapper.html() ).toMatchSnapshot()
    });
})