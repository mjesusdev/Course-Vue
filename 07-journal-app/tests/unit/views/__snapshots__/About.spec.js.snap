// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in About View should run the component correctly 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
